zebrad/src/components/sync/recent_sync_lengths.rs

```diff
@@ -0,0 +1,87 @@
+//! A channel which holds a list of recent syncer response lengths.
+
+use tokio::sync::watch;
+
+#[cfg(test)]
+mod tests;
+
+/// A helper type which holds a list of recent syncer response lengths.
+/// These sync lengths can be used to work out if Zebra has reached the end of the chain.
+///
+/// New lengths are added to the front of the list.
+/// Old lengths are dropped if the list is longer than `MAX_RECENT_LENGTHS`.
+//
+// TODO: disable the mempool if obtain or extend tips return errors?
+#[derive(Debug)]
+pub struct RecentSyncLengths {
+    ///
```

Comment:
I think the documentation for the field is missing here? Or maybe it can just be removed?

Reply:
Added docs in 0e8abedc